src/bosh-director-core/lib/bosh/director/core/templates/job_template_renderer.rb

```diff
@@ -23,6 +24,13 @@ def render(spec)
         spec = remove_unused_properties(spec)
       end
 
+      if @release
```

Comment:
@dpb587-pivotal, thank you for you comment. Actually because of it I found some non-standard behaviour in case of cloud-check related vm recreation. In that case there is no release information so far. Investigating, how I could fix it...